htdocs/app/Migrations/Version20170526212910.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20170526212910 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     * @return void
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf(
+            $this->connection->getDatabasePlatform()->getName() != 'mysql',
+            'Migration can only be executed safely on \'mysql\'.'
+        );
+
+        $this->addSql("
+            UPDATE sys_menu SET href = 'page/impressum' WHERE menustring ='MNU_START_IMPRINT';
```

Comment:
Das musst Du in der [sys_menu.sql](https://github.com/OpencachingDeutschland/oc-server3/blob/development/sql/static-data/sys_menu.sql) fixen. Wenn Du das über eine Migration machst klappt das beim ersten deployment weil nach dem import der static files die Migrations ausgeführt werden. Beim nächsten mal werden aber wieder alle static files importiert und anschließend wird dann keine Migration mehr ausgeführt.